### DISCUSSION TITLE
Setting Local path ?

### DISCUSSION THREAD

**Discussion by LaCourDk** [Natizyskunk/vscode-sftp#523] [Q&A]
if im having 5 SFTP Servers up and running i whant to Have them all Downloaded but Dont whant to Keep Re opening All the Code Spaces ? can i make it so I have a Head Folder 

main Folder 
  Sub folder 1    Connected to server 1 
  Sub Folder 2    connected to server 2 
  Sub Folder 3   connected to server 3

**Comment by jlafitte** [comment]
I wanted the same.  There is a "context" feature which I thought was it, but it's not.  Not even sure how it's different from profiles, the documentation doesn't really explain it's use.
